src/state/apis/portals/portalsApi.ts

```diff
@@ -715,9 +715,19 @@ export const portals = createApi({
             type: DefiType.Staking,
           }
 
+          const lpMetadataUpsertPayload: GetOpportunityMetadataOutput = {
+            byId: {},
+            type: DefiType.LiquidityPool,
+          }
+
           // Populate read only metadata payload
           for (const id in readOnlyMetadata) {
-            stakingMetadataUpsertPayload.byId[id] = readOnlyMetadata[id]
+            const metadata = readOnlyMetadata[id]
+            if (metadata.type === DefiType.LiquidityPool) {
+              lpMetadataUpsertPayload.byId[id] = metadata
+            } else {
+              stakingMetadataUpsertPayload.byId[id] = metadata
+            }
```

Comment:
Ensure we prepare for populating `opportunities.lp.byId`, as we were upserting all into staking umbrella before